cumulus/variables.tf

```diff
@@ -152,6 +152,16 @@ variable "urs_client_password" {
 
 # Optional
 
+variable "urs_tea_client_id" {
+  type = string
+  default = null
+}
+
+variable "urs_tea_client_password" {
+  type = string
+  default = null
+}
```

Comment:
Could you explain to me the use case for having different URS clients for TEA and cumulus?

Reply:
Good question!  This update predates my time at NSIDC, but I believe this is because the dashboard (corresponding to the urs_client_id/password) and the TEA (corresponding to the urs_tea_client_id/password) are two separate EDL applications that have different IDs and passwords.